Documentation/Signoff/Vehicle Lights.md

```diff
@@ -0,0 +1,111 @@
+
+
+
+
+
+      
+## Buildable Schematic
+![image](https://github.com/user-attachments/assets/c47e4d78-0a15-4318-b494-09bd6bab0013)
+
+Figure 1. Car Lights Output Schematic. 
+
+
+![image](https://github.com/user-attachments/assets/b08d7794-4854-4fa5-bc48-4e394aca1a2e)
+
+
+Figure 2. PCB of the LEDs with current limiting resistors in series and mounting holes and screw terminals to connect inputs and any grounds.
+
+![image](https://github.com/user-attachments/assets/2ba29565-6127-4d34-94f4-e0f350534df3)
+
+Figure 3. PCB of the dip switch with CPU input to the switches and screw terminals to connect both the switch output and the input the switch allows through to the different LEDs.
+
+
+![image](https://github.com/user-attachments/assets/10809dc4-c311-4016-81f0-750288931a8e)
+
+
+Figure 4. 3D model of LEDs.
+
+![image](https://github.com/user-attachments/assets/57745358-546d-46a9-8ac4-165cf79125cd)
+
+
+Figure 5. 3D model of Dip SW 
+
+## Analysis 
+
+### Select Line / Enable Switches
+![image](https://github.com/user-attachments/assets/e414383f-0a64-4c0d-b218-7a5033957ce9)
+
+
+
+Figure 6. Select input / Enable Switches
+
+Dip switches will be used to control the input sent to the LEDs. The reason behind using dip switches is to give the user a better learning experience by allowing them to slide the switch up representing a 1 and slide down representing 0, they will also be able to visualize it. Based on the user input through the switches the output of this subsystem will be affected accordingly. The switches will be configured to the Arduino 2560 within the input system and coded so the state will be read by the Arduino 2560 which is the master CPU. The CPU will process that data and send it out of a digital pin to the headlights. Below is a truth table that will represent the dip switches functionality. The dip switches will represent select lines to the coded functionality of a multiplexer. The dip switches will take in data from the Master CPU and to do that the switches will have solder wire holes so that the wires from the CPU can be soldered into the holes that are traced over to the input side of the switch. This is represented in Figure 5. as j4, j5, and j7. The Dip switch pins 4, 5, and 6 will all be connected into an 8-level terminal block where pin 6 needs to go to every LED because that will be the ON/OFF switch. This means pin 6 will have 4 of those terminals that will all be traced together, which is shown in the 3D model in Figure 5., and wire will be from those terminals to each current limiting resistor as needed. Pin 4 will have 2 of those terminals that are traced together since it will send its output to two LEDs to represent the turn signals. Pin 5 will also have two terminals traced together so it can represent the other turn signals.
+
```

Comment:
Why are we using dip switches when we have a microprocessor and HMI? Would it not be easier to connect each resistor-LED network to a separate pin on the microprocessor?